lib/vtls/schannel.c

```diff
@@ -2318,40 +2296,38 @@ schannel_recv(struct Curl_cfilter *cf, struct Curl_easy *data,
         backend->recv_sspi_close_notify = true;
         if(!backend->recv_connection_closed)
           backend->recv_connection_closed = true;
-        infof(data,
-              "schannel: server close notification received (close_notify)");
+        CURL_TRC_CF(data, cf,
+                    "server close notification received (close_notify)");
         goto cleanup;
       }
     }
     else if(sspi_status == SEC_E_INCOMPLETE_MESSAGE) {
       backend->encdata_is_incomplete = true;
       if(!*err)
         *err = CURLE_AGAIN;
-      infof(data, "schannel: failed to decrypt data, need more data");
+      CURL_TRC_CF(data, cf, "incomplete message, EAGAIN");
```

Comment:
The condition above indicates that this might happen for errors other than EAGAIN, and then this message gets misleading. How about now showing the message at all for the EAGAIN case since that looks like a more "normal" code path and not really something to mention in the verbose.

Reply:
Will remove it.